assignments/this.js

```diff
@@ -1,10 +1,10 @@
 /* The for principles of "this";
  * in your own words. explain the four principle for the "this" keyword below.
  *
- * 1.
- * 2.
- * 3.
- * 4.
+ * 1. Implicit this used in methods
+ * 2. Explicit this used with call(), apply(), bind()
+ * 3. Global this pointing at Window or undefined
+ * 4. This used in object created with the "new" keyword
```

Comment:
'new' keyword makes a function call a constructor call. When a function is invoked with constructor call
a new object is created (constructed) and set as 'this' binding for that function call.